_component_design/breadcrumbs.md

```diff
@@ -4,11 +4,11 @@ parent: Navigation
 layout: component
 category: Components
 usage: >
-  Breadcrumbs are an inline visual representation of page relationships and hierarchy. They provide a user with visual reinforcement of the current location within the application hierarchy, and allow for quick movement up the hierarchy.
+  Breadcrumbs are a linear navigation scheme that shows the users' current location in a website or application. A user can click  breadcrumb elements to navigate up the site hierarchy to previously visited pages.
```

Comment:
BC is a page navigation component that shows the users' current location and helps with wayfinding and navigating up the hierarchical chain.